maistra/ci/post-submit.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+set -x
+
+GCS_PROJECT=${GCS_PROJECT:-maistra-prow-testing}
+ARTIFACTS_GCS_PATH=${ARTIFACTS_GCS_PATH:-gs://maistra-prow-testing/proxy}
+
+gcloud auth activate-service-account --key-file="${GOOGLE_APPLICATION_CREDENTIALS}"
+gcloud config set project "${GCS_PROJECT}"
+
+# Fix path to the vendor deps
+sed -i "s|=/work/|=$(pwd)/|" maistra/bazelrc-vendor
+
+ARCH=$(uname -p)
+if [ "${ARCH}" = "ppc64le" ]; then
+  ARCH="ppc"
+fi
+
+# Build
+bazel build \
+  --config=release \
+  --config=${ARCH} \
+  //src/envoy:envoy_tar
+
+# Copy binary to GCS
+SHA="$(git rev-parse --verify HEAD)"
+gsutil cp bazel-bin/src/envoy/envoy_tar.tar.gz "${ARTIFACTS_GCS_PATH}/envoy-alpha-${SHA}.tar.gz"
+
+# Workaround WASM limitations
+gsutil cp "${ARTIFACTS_GCS_PATH}/metadata_exchange.wasm" "${ARTIFACTS_GCS_PATH}/metadata_exchange-${SHA}.wasm"
+gsutil cp "${ARTIFACTS_GCS_PATH}/stats.wasm" "${ARTIFACTS_GCS_PATH}/stats-${SHA}.wasm"
```

Comment:
I suppose we will have to create these manually? Because they're not present right now, so I'm hesitant to merge this

Reply:
You are right. I manually copied those files on my personal bucket but forgot to copy them using maistra account. I'll do that now.

Reply:
@dgn done.

Reply:
thank you!